IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowseTests.cs

```diff
@@ -184,6 +184,66 @@ public void Filters_should_behave_as_expected()
                 });
         }
 
+        [TestCase("", "file.txt", false)]
+        [TestCase("", "file with spaces.txt", false)]
+        [TestCase("Dir with spaces", "file.txt", false)]
+        [TestCase("Dir with spaces", "file with spaces.txt", false)]
+        [TestCase("", "file.txt", true)]
+        [TestCase("", "file with spaces.txt", true)]
+        [TestCase("Dir with spaces", "file.txt", true)]
+        [TestCase("Dir with spaces", "file with spaces.txt", true)]
```

Comment:
> `fileHistoryFileName` comes from `arg[2]`. On a command prompt `foo` or `"foo"` are treated the same - `foo`. We don't supply `"\"foo\""` on a command line and; if a user does it explicitly - then it's user's doing.

> I considered adding a method that would quote if no quotes present, however, it would require a deeper investigation, which I don't have time for.

These testcases try to do what we don't want to support... and fail.

Reply:
Yes, I reverted those. For now these are good enough.